<!DOCTYPE html>
<html>
<head>
  <title>Network biology overview</title>
  <meta charset="utf-8">
  <meta name="description" content="Network biology overview">
  <meta name="author" content="Keith Hughitt">
  <meta name="generator" content="slidify" />
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta http-equiv="X-UA-Compatible" content="chrome=1">
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/default.css" media="all" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/phone.css" 
    media="only screen and (max-device-width: 480px)" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/slidify.css" >
  <link rel="stylesheet" href="libraries/highlighters/highlight.js/css/tomorrow.css" />
  <base target="_blank"> <!-- This amazingness opens all links in a new tab. -->  <link rel=stylesheet href="./assets/css/ribbons.css"></link>

  
  <!-- Grab CDN jQuery, fall back to local if offline -->
  <script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>
  <script>window.jQuery || document.write('<script src="libraries/widgets/quiz/js/jquery.js"><\/script>')</script> 
  <script data-main="libraries/frameworks/io2012/js/slides" 
    src="libraries/frameworks/io2012/js/require-1.0.8.min.js">
  </script>
  
  

</head>
<body style="opacity: 0">
  <slides class="layout-widescreen">
    
    <!-- LOGO SLIDE -->
        <slide class="title-slide segue nobackground">
  <hgroup class="auto-fadein">
    <h1>Network biology overview</h1>
    <h2></h2>
    <p>Keith Hughitt<br/></p>
  </hgroup>
    <a href="https://github.com/khughitt/slidify-network-biology/zipball/gh-pages" class="example">
     Download
    </a>
  <article></article>  
</slide>
    

    <!-- SLIDES -->
    <slide class="" id="slide-1" style="background:;">
  <article data-timings="">
    <!-- Custom Styles -->

<style type='text/css'>
    slides > slide {
        height: 800px;
        margin-top: -400px;
    }
    img {
        max-height: 560px;
        max-width: 964px;
    }
    slide a {border-bottom: none;}
    .references li { font-size: 14px; }
</style>

<!-- Custom JavaScript -->

<script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>

<script type='text/javascript'>
$(function() {
    $("p:has(img)").addClass('centered');
});
</script>

<h2>The Big Picture</h2>

<h3>Why networks?</h3>

<ul>
<li>For most of the history of biology, a reductionist approach has been used to
understand different parts of an ecosystem, an organism, a cell, etc.</li>
<li>This has gotten us pretty far, but you can only understand so much by looking
at things piecemeal...</li>
<li>It is only by looking at how all of the parts work together (as a &quot;system&quot;)
that we can truly understand how something works.</li>
<li>Once the quality of networks are sufficient, they can also be used to make
predictions.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-2" style="background:;">
  <hgroup>
    <h2>The problem...</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/cars_golf_volkswagen_parts_desktop_1440x900_hd-wallpaper-44957.jpg" alt="car parts"></p>

<div style='float: right; width: 38%; font-size: 12px;'>source: http://onlyhdwallpapers.com/wallpaper/cars_golf_volkswagen_parts_desktop_1440x900_hd-wallpaper-44957.jpg</div>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="segue dark" id="slide-3" style="background:;">
  <hgroup>
    <h2>Types of Biological Networks</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-4" style="background:;">
  <hgroup>
    <h2>Types of Biological Networks</h2>
  </hgroup>
  <article data-timings="">
    <ol>
<li>Cell signalling</li>
<li>Metabolic interactions</li>
<li>Protein-protein interaction</li>
<li>Co-expression</li>
<li>Transcription factor binding</li>
<li>Protein-phosphorylation</li>
<li>Genetic interaction</li>
<li>Gene regulatory</li>
</ol>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-5" style="background:;">
  <hgroup>
    <h2>Types of data used to construct biological networks</h2>
  </hgroup>
  <article data-timings="">
    <ol>
<li>Flow cytometry</li>
<li>Expression (Microarray, RNA-Seq)</li>
<li>TF binding (ChIP-chip, ChIP-Seq, etc)</li>
<li>Protein interaction (Y2H, TAP-MS)</li>
<li>Metabolite concentrations (NMR, LC/MS, etc)</li>
</ol>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-6" style="background:;">
  <hgroup>
    <h2>Transcription factor binding networks</h2>
  </hgroup>
  <article data-timings="">
    <h3>Overview</h3>

<ul>
<li>Directed.</li>
<li>TF binding data provides information of gene <span class='red'>regulatory 
relationships.</span></li>
<li>Which genes (transcription factors) regulate which other genes?</li>
</ul>

<h3>Data Sources</h3>

<ul>
<li>Protein-DNA binding (ChIP)</li>
<li>Expression (microarray, RNA-Seq)</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-7" style="background:;">
  <hgroup>
    <h2>Transcription factor binding networks</h2>
  </hgroup>
  <article data-timings="">
    <h3>Chromatin immunoprecipitation (ChIP)</h3>

<ul>
<li>Provides information on <em>in vivo</em> protein-DNA binding associations.</li>
<li>Antibodies targeting specific proteins are used to pull-down proteins in
a sample and then associated DNA fragments measured.</li>
<li>E.g. <span class='blue'>ChIP-chip</span> and <span class='blue'>ChIP-Seq
</span>.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-8" style="background:;">
  <hgroup>
    <h2>Transcription factor binding networks</h2>
  </hgroup>
  <article data-timings="">
    <h3>Combining with expression data</h3>

<ul>
<li>Another approach is to combine ChIP data with expression data such as
microarray data.

<ul>
<li>Expression profile correlations helps strengthen the relationship between
TF&#39;s and the genes they regulate.</li>
<li>ChIP provides the directionality in the relationship.</li>
<li>See <em>De Smet &amp; Marchal</em> (2010) for a review.</li>
</ul></li>
</ul>

<h3>Challenges</h3>

<ul>
<li>Not all binding sites are near genes they regulate; may exist near unrelated
genes.</li>
<li>Binding sites may be either redundant or not functionally active.</li>
<li>Role of binding site may be modified by conditions.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-9" style="background:;">
  <hgroup>
    <h2>Transcription factor binding networks</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/nbt890-F1.gif" alt="tf network">
<span class='caption'>Bar-Joseph et al. (2003)</span></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-10" style="background:;">
  <hgroup>
    <h2>Protein-protein interaction networks</h2>
  </hgroup>
  <article data-timings="">
    <!--<iframe width="640" height="480" src="//www.youtube.com/embed/Ms_ehUVvKKk"
frameborder="0" allowfullscreen></iframe>-->

<p>Interleukin-1 binding to its receptor on a cell surface, created from
structural data (Youtube).</p>

<p><a href="http://www.youtube.com/watch?v=Ms_ehUVvKKk"><img src="http://img.youtube.com/vi/Ms_ehUVvKKk/0.jpg" alt="Protein interaction animation"></a></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-11" style="background:;">
  <hgroup>
    <h2>Protein-protein interaction networks</h2>
  </hgroup>
  <article data-timings="">
    <h3>Overview</h3>

<ul>
<li>Protein-protein interaction (PPI) networks describe direct <span class='red'>
physical interactions</span> between proteins in a cell.</li>
<li>This includes:

<ul>
<li>Protein complexes</li>
<li>Transiently interacting proteins</li>
</ul></li>
<li>Undirected</li>
<li>The complete set of protein interactions in a cell is sometimes referred to
as the <span class='red'>interactome</span></li>
</ul>

<h3>Data Sources</h3>

<ul>
<li><span class='blue'>Yeast two-hybrid (Y2H)</span></li>
<li><span class='blue'>Tandem affinity purification coupled to mass spectometry
(TAP-MS)</span></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-12" style="background:;">
  <hgroup>
    <h2>Protein-protein interaction networks</h2>
  </hgroup>
  <article data-timings="">
    <h3>Challenges</h3>

<ul>
<li>Y2H data is collected <em>in vitro</em> and may not provide an accurate 
representation of the catalog of interactions that actually take place in a 
cell under normal conditions:

<ul>
<li>Real interactions may not hold up and will be missed</li>
<li>Complexes that do not form <em>in vivo</em> may be found <em>in vitro</em></li>
</ul></li>
<li>Affinity purification can be noisy (protein contamination.)</li>
<li>Overlap between Y2H is not 100%.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-13" style="background:;">
  <hgroup>
    <h2>Protein-protein interaction networks</h2>
  </hgroup>
  <article data-timings="">
    <p>Yeast Membrane Proteins Interaction Network (Babu lab)</p>

<p><img src="assets/img/Figure%203-new.png" alt="Yeast PPI"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-14" style="background:;">
  <hgroup>
    <h2>Metabolic Networks</h2>
  </hgroup>
  <article data-timings="">
    <h3>Definition</h3>

<blockquote>
<p>A metabolic network is the complete set of metabolic and physical processes
that determine the physiological and biochemical properties of a cell. As
such, these networks comprise the chemical reactions of metabolism, the
metabolic pathways, as well as the regulatory interactions that guide these
reactions. (Wikipedia)</p>
</blockquote>

<h3>Overview</h3>

<ul>
<li>Directed</li>
<li>Nodes: metabolites</li>
<li>Edges: biochemical reactions</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-15" style="background:;">
  <hgroup>
    <h2>Metabolic Networks</h2>
  </hgroup>
  <article data-timings="">
    <h3>KEGG</h3>

<p><img src="assets/img/kegg.gif" alt="KEGG"></p>

<div style='float: right; width: 38%; font-size: 12px;'>source: http://www.cs.cmu.edu/~blmt/Seminar/SeminarMaterials/IntroMolBasDisease.html</div>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-16" style="background:;">
  <hgroup>
    <h2>Metabolic Networks</h2>
  </hgroup>
  <article data-timings="">
    <p>Another view of the human metabolome...</p>

<p><img src="assets/img/pathway-1b.png" alt="human metabolome"></p>

<div style='float: right; width: 38%; font-size: 12px;'>source: http://nuit-blanche.blogspot.com/2011/11/instances-of-null-spaces-can.html</div>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-17" style="background:;">
  <hgroup>
    <h2>Metabolic Networks</h2>
  </hgroup>
  <article data-timings="">
    <h3>Flux balance analysis (FBA)</h3>

<ul>
<li><em>In silico</em> approach commonly used for metabolic network reconstruction.
Uses known stoichiometric information about metabolites for an organism.</li>
<li>Assumes that organism is making optimal use of resources.</li>
<li>Network parameters are chosen such that the steady-state biomass output or
growth rate is maximized.</li>
<li>Uses <span class='blue'>linear programming</span>.</li>
<li>Has been used successfully to make some predictions about affects of changes
on a metabolic network.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-18" style="background:;">
  <hgroup>
    <h2>Co-expression Networks</h2>
  </hgroup>
  <article data-timings="">
    <h3>Overview</h3>

<ul>
<li>Undirected</li>
<li>Nodes: gene products</li>
<li>Edges: co-expression relationships</li>
</ul>

<h3>Data sources</h3>

<ul>
<li>Microarray</li>
<li>RNA-Seq</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-19" style="background:;">
  <hgroup>
    <h2>Co-expression Networks</h2>
  </hgroup>
  <article data-timings="">
    <h3>Approaches</h3>

<ul>
<li>Bayesian Networks (Friedman et al., 2000)</li>
<li>Relevance Networks (Butte &amp; Kohane, 2000)</li>
<li>Correlation Networks (Stuart et al. 2003; Zhang &amp; Horvath, 2005)</li>
<li>Gaussian Graphical Models (Schäfer &amp; Strimmer, 2005)</li>
<li>Canonical correlation analysis (Hong et al. 2013)</li>
</ul>

<h3>Challenges</h3>

<ul>
<li>Interpretation of co-expression networks is not as straight-forward as some
of the other types of networks; gene products may be co-expressed because
they are regulated by same TF, because they interact closely, etc.</li>
<li>How correlated should expression profiles be before gene products are
considered co-expressed?</li>
<li>Co-expression status can be highly variable with time and condition.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-20" style="background:;">
  <hgroup>
    <h2>Co-expression Networks</h2>
  </hgroup>
  <article data-timings="">
    <p>T. cruzi Co-expression network (Constructed via WGCNA)</p>

<p><img src="assets/img/RNABP.jpg" alt="T. cruzi Co-expression network"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="segue dark" id="slide-21" style="background:;">
  <hgroup>
    <h2>Biological network analysis</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-22" style="background:;">
  <hgroup>
    <h2>So, I built a network... now what?</h2>
  </hgroup>
  <article data-timings="">
    <p>It&#39;s usually not enough to just build a network and print a pretty picture --
or at least, it shouldn&#39;t be enough -- instead a useful network should provide
some information about the system being investigated which is not immediately
available by looking at its component in isolation.</p>

<p>Depending on the type of network being investigated, different approaches may
be adopted:</p>

<ul>
<li>Module detection</li>
<li>Annotation (e.g. <span class='blue'>&quot;guilt by association&quot;</span>)</li>
<li>Impact of disease, etc. to underlying biochemical functioning</li>
<li>Targeted drug development</li>
<li>Comparative network analysis</li>
<li>Network integration</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="references" id="slide-23" style="background:;">
  <hgroup>
    <h2>References</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>Ziv Bar-Joseph, Georg K Gerber, Tong Ihn Lee, Nicola J Rinaldi, Jane Y Yoo, François Robert, D Benjamin Gordon, Ernest Fraenkel, Tommi S Jaakkola, Richard A Young, David K Gifford,   (2003) Computational Discovery of Gene Modules And Regulatory Networks.  <em>Nature Biotechnology</em>  <strong>21</strong>  1337-1342  <a href="http://dx.doi.org/10.1038/nbt890">10.1038/nbt890</a></li>
<li>Javier De Las Rivas, Celia Fontanillo, Fran Lewitter,   (2010) Protein–Protein Interactions Essentials: Key Concepts to Building And Analyzing Interactome Networks.  <em>Plos Computational Biology</em>  <strong>6</strong>  e1000807-NA  <a href="http://dx.doi.org/10.1371/journal.pcbi.1000807">10.1371/journal.pcbi.1000807</a></li>
<li>Riet De Smet, Kathleen Marchal,   (2010) Advantages And Limitations of Current Network Inference Methods.  <em>Nature Reviews Microbiology</em>  <a href="http://dx.doi.org/10.1038/nrmicro2419">10.1038/nrmicro2419</a></li>
<li>Nir Friedman, Michal Linial, Iftach Nachman, Dana Pe&#39;er,   (2000) Using Bayesian Networks to Analyze Expression Data.  <em>Journal of Computational Biology</em>  <strong>7</strong>  601-620  <a href="http://dx.doi.org/10.1089/106652700750050961">10.1089/106652700750050961</a></li>
<li>S. Hong, X. Chen, L. Jin, M. Xiong,   (2013) Canonical Correlation Analysis For Rna-Seq co-Expression Networks.  <em>Nucleic Acids Research</em>  <strong>41</strong>  e95-e95  <a href="http://dx.doi.org/10.1093/nar/gkt145">10.1093/nar/gkt145</a></li>
<li>Peter J. Park,   (2009) Chip–Seq: Advantages And Challenges of A Maturing Technology.  <em>Nature Reviews Genetics</em>  <strong>10</strong>  669-680  <a href="http://dx.doi.org/10.1038/nrg2641">10.1038/nrg2641</a></li>
<li>Eytan Ruppin, Jason A Papin, Luis F de Figueiredo, Stefan Schuster,   (2010) Metabolic Reconstruction, Constraint-Based Analysis And Game Theory to Probe Genome-Scale Metabolic Networks.  <em>Current Opinion in Biotechnology</em>  <strong>21</strong>  502-510  <a href="http://dx.doi.org/10.1016/j.copbio.2010.07.002">10.1016/j.copbio.2010.07.002</a></li>
<li>unknown unknown,   (unknown) Unknown.  <em>Unknown</em></li>
<li>Bin Zhang, Steve Horvath,   (2005) A General Framework For Weighted Gene co-Expression Network Analysis.  <em>Statistical Applications in Genetics And Molecular Biology</em>  <strong>4</strong>  <a href="http://dx.doi.org/10.2202/1544-6115.1128">10.2202/1544-6115.1128</a></li>
<li><p>X. Zhu, M. Gerstein, M. Snyder,   (2007) Getting Connected: Analysis And Principles of Biological Networks.  <em>Genes &amp; Development</em>  <strong>21</strong>  1010-1024  <a href="http://dx.doi.org/10.1101/gad.1528707">10.1101/gad.1528707</a></p></li>
<li><p>Butte, a J., &amp; Kohane, I. S. (2000). Mutual information relevance networks:
functional genomic clustering using pairwise entropy measurements. Pacific
Symposium on Biocomputing. Pacific Symposium on Biocomputing, 418–29. Retrieved
from <a href="http://www.ncbi.nlm.nih.gov/pubmed/10902190">http://www.ncbi.nlm.nih.gov/pubmed/10902190</a></p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

    <slide class="backdrop"></slide>
  </slides>
  <div class="pagination pagination-small" id='io2012-ptoc' style="display:none;">
    <ul>
      <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=1 title=''>
         1
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=2 title='The problem...'>
         2
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=3 title='Types of Biological Networks'>
         3
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=4 title='Types of Biological Networks'>
         4
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=5 title='Types of data used to construct biological networks'>
         5
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=6 title='Transcription factor binding networks'>
         6
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=7 title='Transcription factor binding networks'>
         7
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=8 title='Transcription factor binding networks'>
         8
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=9 title='Transcription factor binding networks'>
         9
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=10 title='Protein-protein interaction networks'>
         10
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=11 title='Protein-protein interaction networks'>
         11
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=12 title='Protein-protein interaction networks'>
         12
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=13 title='Protein-protein interaction networks'>
         13
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=14 title='Metabolic Networks'>
         14
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=15 title='Metabolic Networks'>
         15
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=16 title='Metabolic Networks'>
         16
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=17 title='Metabolic Networks'>
         17
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=18 title='Co-expression Networks'>
         18
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=19 title='Co-expression Networks'>
         19
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=20 title='Co-expression Networks'>
         20
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=21 title='Biological network analysis'>
         21
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=22 title='So, I built a network... now what?'>
         22
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=23 title='References'>
         23
      </a>
    </li>
  </ul>
  </div>  <!--[if IE]>
    <script 
      src="http://ajax.googleapis.com/ajax/libs/chrome-frame/1/CFInstall.min.js">  
    </script>
    <script>CFInstall.check({mode: 'overlay'});</script>
  <![endif]-->
</body>
  <!-- Load Javascripts for Widgets -->
  
  <!-- MathJax: Fall back to local if CDN offline but local image fonts are not supported (saves >100MB) -->
  <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        inlineMath: [['$','$'], ['\\(','\\)']],
        processEscapes: true
      }
    });
  </script>
  <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
  <!-- <script src="https://c328740.ssl.cf1.rackcdn.com/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  </script> -->
  <script>window.MathJax || document.write('<script type="text/x-mathjax-config">MathJax.Hub.Config({"HTML-CSS":{imageFont:null}});<\/script><script src="libraries/widgets/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><\/script>')
</script>
<!-- LOAD HIGHLIGHTER JS FILES -->
  <script src="libraries/highlighters/highlight.js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>
  <!-- DONE LOADING HIGHLIGHTER JS FILES -->
   
  </html>
